cmd/fluent-watcher/fluentbit/Dockerfile

```diff
@@ -1,13 +1,19 @@
-FROM golang:1.20.4-alpine3.16 as buildergo
+ARG FLUENTBIT_BASE
+FROM ghcr.io/verrazzano/verrazzano-base:v1.0.0-20230327155846-4653b27 as buildergo
```

Comment:
At other place you are using ```ghcr.io/verrazzano/verrazzano-base:latest``` why so ?

Reply:
I copied this from what we have in 2.2.0, actually `latest` tag isn't an option with verrazzano-base images. I can get rid of `latest` references in a follow up PR.

Reply:
I mean in the same PR above, also is :v1.0.0-20230327155846 the latest available

Reply:
Will point to the latest version

Reply:
Pointed to the latest available base image and removed the reference of `latest` tag.